### ISSUE TITLE
OpenSource Programs : Winter Of Code

### ISSUE THREAD

**Issue by Kajal13081** [docsopensource/docsopensource.github.io#7]
I would like to add a documentation about one more open source program woc by Gdsc Nsec.
Here is the website:
https://winterofcode.com/

**Comment by Kajal13081** [comment]
@Aryamanz29 Can you assign me this issue please.

**Comment by Aryamanz29** [comment]
Yes , Go ahead @Kajal13081👍 , Let me know if you face any problem.

**Comment by Kajal13081** [comment]
Sure, Thank You
